core/src/main/java/hudson/Functions.java

```diff
@@ -2455,4 +2455,16 @@ static String guessIcon(String iconGuess, String rootURL) {
     public static String generateItemId() {
         return String.valueOf(Math.floor(Math.random() * 3000));
     }
+
+    @Restricted(NoExternalUse.class)
+    public static String translateModifierKeysForUsersPlatform(String keyboardShortcut) {
```

Comment:
As the Jellydoc is misleading, this would benefit from a Javadoc explaining use.

Reply:
Added.